pulsar-broker/src/main/java/org/apache/pulsar/PulsarStandaloneStarter.java

```diff
@@ -76,6 +81,12 @@
     @Parameter(names = { "--only-broker" }, description = "Only start Pulsar broker service (no ZK, BK)")
     private boolean onlyBroker = false;
 
+    @Parameter(names = {"-rfw", "--run-function-worker"}, description = "Run function worker with Broker")
```

Comment:
`--run-functions-worker` 

Also, we should have a key in `broker.conf` to do the same

Reply:
will add one.